project_national/fact_sheets_category_6.yml

```diff
@@ -0,0 +1,752 @@
+      - &ducted_ASHP_EnergyStar_exist_backup_shared_ducts_80_FO
+        option: HVAC Heating Efficiency|Dual-Fuel ASHP, SEER 16, 9.2 HSPF, Integrated Backup, 80% AFUE Fuel Oil, 5F-40F switchover band
+        apply_logic: &dual-fuel_80pctAFUE_FO_furnace_apply_logic
+          - HVAC Has Ducts|Yes
+          - HVAC Heating Efficiency|Fuel Furnace, 80% AFUE
+          - HVAC Heating Type And Fuel|Fuel Oil Fuel Furnace        
+        costs: *costs_ashp_ducted_SEER_16_9-2_HSPF
+        lifetime: *lifetime_ashp_ducted_SEER_16_9-2_HSPF
+      - &ducted_ASHP_EnergyStar_exist_backup_shared_ducts_92-5_FO
+        option: HVAC Heating Efficiency|Dual-Fuel ASHP, SEER 16, 9.2 HSPF, Integrated Backup, 92.5% AFUE Fuel Oil, 5F-40F switchover band
+        apply_logic: &dual-fuel_92-5pctAFUE_FO_furnace_apply_logic
+          - HVAC Has Ducts|Yes
+          - HVAC Heating Efficiency|Fuel Furnace, 92.5% AFUE
+          - HVAC Heating Type And Fuel|Fuel Oil Fuel Furnace        
+        costs: *costs_ashp_ducted_SEER_16_9-2_HSPF
+        lifetime: *lifetime_ashp_ducted_SEER_16_9-2_HSPF
+
+
+      - &HP_cooling_efficiency_ducted
+        option: HVAC Cooling Efficiency|Ducted Heat Pump
+        apply_logic:
+          - or:
+            - *dualfuel_ASHP_no_ducted_furnace_apply_logic
+            - *dual-fuel_60pctAFUE_NG_furnace_apply_logic
+            - *dual-fuel_76pctAFUE_NG_furnace_apply_logic
+            - *dual-fuel_80pctAFUE_NG_furnace_apply_logic
+            - *dual-fuel_92-5pctAFUE_NG_furnace_apply_logic
+            - *dual-fuel_60pctAFUE_propane_furnace_apply_logic
+            - *dual-fuel_76pctAFUE_propane_furnace_apply_logic
+            - *dual-fuel_80pctAFUE_propane_furnace_apply_logic
+            - *dual-fuel_92-5pctAFUE_propane_furnace_apply_logic
+            - *dual-fuel_60pctAFUE_FO_furnace_apply_logic
+            - *dual-fuel_76pctAFUE_FO_furnace_apply_logic
+            - *dual-fuel_80pctAFUE_FO_furnace_apply_logic
+            - *dual-fuel_92-5pctAFUE_FO_furnace_apply_logic
+
+
+
+
+      - &100_percent_conditioned_ducted
+        option: HVAC Cooling Partial Space Conditioning|100% Conditioned
+        apply_logic:
+          - or:
+            - *dualfuel_ASHP_no_ducted_furnace_apply_logic
+            - *dual-fuel_60pctAFUE_NG_furnace_apply_logic
+            - *dual-fuel_76pctAFUE_NG_furnace_apply_logic
+            - *dual-fuel_80pctAFUE_NG_furnace_apply_logic
+            - *dual-fuel_92-5pctAFUE_NG_furnace_apply_logic
+            - *dual-fuel_60pctAFUE_propane_furnace_apply_logic
+            - *dual-fuel_76pctAFUE_propane_furnace_apply_logic
+            - *dual-fuel_80pctAFUE_propane_furnace_apply_logic
+            - *dual-fuel_92-5pctAFUE_propane_furnace_apply_logic
+            - *dual-fuel_60pctAFUE_FO_furnace_apply_logic
+            - *dual-fuel_76pctAFUE_FO_furnace_apply_logic
+            - *dual-fuel_80pctAFUE_FO_furnace_apply_logic
+            - *dual-fuel_92-5pctAFUE_FO_furnace_apply_logic
+
```

Comment:
These are the only two global changes? 100% conditioned and HVAC Cooling Efficiency?